pkg/util/util.go

```diff
@@ -26,6 +26,41 @@ import (
 	"golang.org/x/mod/modfile"
 )
 
+// ResultTypeSupportedVerbs is a list of verb types that supports overriding the
+// resulting type.
+var ResultTypeSupportedVerbs = []string{
```

Comment:
How did we come to these lists? What does it mean to override the result type? If we can override on create/update, why not updatestatus?

Reply:
Taken from here: https://github.com/kubernetes/code-generator/blob/7d977b3e5454e47ec6b903242616e9ea62644226/cmd/client-gen/generators/util/tags.go#L63-L89. It means that the users cannot override the result type for verbs like `Watch` etc.